### PR TITLE
Fix traj parsing

### PR DESCRIPTION
**Pull request by ceriottm**
@EliaStocco  let's fix here the parser, before moving to the consolidation mechanism.

### CODE REVIEW

**Comment by ceriottm** [comment]
given that it looks like we've kind of lost momentum following the hackaton, I'd like to merge this, given that the current code is buggy. I would also then make a point release so we can start using and improving the scripting interface, ffdirect, the atom constraining, and all the other nice add-ons that were implemented.

File: ipi/utils/parsing.py [hunk 1]
```diff
@@ -219,13 +219,37 @@ def read_trajectory(
...
+                    frame.positions *= 0.0  # trajectory does NOT contain position data!
+                    frame.arrays[what] = ret["atoms"].q.reshape((-1, 3))
+                    if what in [
+                        "forces",
+                        "forces_spring",
+                        "forces_component",
+                        "forces_sc",
+                        "Eforces",
+                        "f_centroid",
+                        "forces_component_raw",
```
Comment (mahrossi) [inline ipi/utils/parsing.py:241]: so every time we add a new position keyword of force keyword we have to hand-modify this list, yes? maybe worth adding a disclaimer about this in the file.

**Review by mahrossi — APPROVED** [review]

File: ipi/utils/parsing.py (hunk 1 above)
Comment (ceriottm) [reply ipi/utils/parsing.py:241]: Yes, I don't see an easy way around it unfortunately.

Comment (ceriottm) [reply ipi/utils/parsing.py]: Actually I didn't think hard enough. Thanks for spotting this, there as indeed a way to avoid the manual mess.

**Review by mahrossi — APPROVED** [review]

**Comment by mahrossi** [comment]
Amazing, this looks way better. I will merge.

**Pull request merged**